config/application.rb

```diff
@@ -67,6 +67,15 @@ class Application < Rails::Application
         )
       config.registry_public_key = config.registry_private_key.public_key
       config.access_policies = '/etc/rmt/access_policies.yml'
+
+      config.cache_config_file = '/var/lib/rmt/rmt-cache-trim.sh'
+      registry_cache_expiration_time = Settings[:registry].try(:token_expiration) || 28800 # 8.hours.to_i
```

Comment:
We use `registry_cache_expiration_time` to set a value for a variable that is called `REGISTRY_CACHE_EXPIRY_MINUTES`, so that means, since the unit `minutes` is embedded in the key name that we have 28800 minutes of cache time which is 480 hours and as such does not meet the specification. The implementation details specify the value as `$VALUE_FROM_CONFIG_FILE_480_BY_DEFAULT`

We might also be want to be more explicit with the configuration value. Either name the setting as `token_expiration_minutes` or allow the user to specify a postfix of `s`, `m`, `h` and then convert to the units we expect.